indico/modules/auth/templates/users/accounts.html

```diff
@@ -0,0 +1,150 @@
+                        <span id="no-local-acccount">
+                            {% trans %}You don't have a local account yet.{% endtrans %}
+                            <a href="#" class="js-show-form">{% trans %}Create one{% endtrans %}</a>.
+                        </span>
+                        <div id="credentials-form" class="hidden">
+                            {{ form_header(orientation='vertical', classes='no-block-padding') }}
+                            {{ form_rows(form) }}
+                            {% call form_footer() %}
+                                <input class="i-button big highlight" type="submit"
+                                    value="{% trans %}Create account{% endtrans %}">
+                                <a href="#" class="i-button big js-hide-form">
+                                    {% trans %}Cancel{% endtrans %}
+                                </a>
+                            {% endcall %}
+                            <script>
+                                validatePasswordConfirmation('#password', '#confirm_password');
+                            </script>
+                        </div>
+                    {% else %}
+                        {{ form_header(orientation='vertical', classes='no-block-padding') }}
+                        {{ form_rows(form) }}
+                        {% call form_footer() %}
+                            <input class="i-button big highlight" type="submit" value="{% trans %}Modify credentials{% endtrans %}">
+                        {% endcall %}
+                        <script>
+                            validatePasswordConfirmation('#new_password', '#confirm_new_password');
+                        </script>
+                    {% endif %}
+                    {% if user.secondary_local_identities %}
+                        <p>
+                        </p>
+                        <ul id="local-secondary" class="group-list with-buttons">
+                            <li>
+                                {% trans %}Other local accounts used in the past.{% endtrans -%}
+                                <a href="#" class="js-toggle-secondary-accounts">
+                                    {% trans %}Show{% endtrans %}</a>
+                                <a href="#" class="js-toggle-secondary-accounts hidden">
+                                    {% trans %}Hide{% endtrans %}</a>
+                            </li>
+                            {% for identity in user.secondary_local_identities %}
+                                <li class="hidden">
+                                    <span class="account-identity">
+                                        <i class="icon-user"></i>
+                                        <span>{{ identity.identifier }}</span>
+                                    </span>
+                                    <a href="{{ url_for('auth.remove-account', identity) }}" class="i-button right">
```

Comment:
You can use these data-attributes to have an i-button that performs a POST request (using a form that's generated in JS on the fly - no need to write extra javascript/html besides that):

        <button class="i-button right"
                data-href="{{ url_for('.remove_account', identity) }}"
                data-method="post"
                data-title="{% trans %}Remove account{% endtrans %}"
                data-confirm="{% trans %}Do you really want to remove this account?{% endtrans %}">